src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -375,7 +375,7 @@ internal void ApplyReplacementText(string replacementText, bool propagateEditImm
         {
             AssertIsForeground();
             VerifyNotDismissed();
-            this.ReplacementText = _renameInfo.GetFinalSymbolName(replacementText);
```

Comment:
💭 It's possible this was in place due to the difference between attribute names at the declaration and use site. For example, at the declaration site `InAttribute` appears as `InAttribute` (includes the suffix). However, when the attribute is used it appears as just `[In]` (without the suffix). We'll need to verify that the tests cover both cases.

Reply:
I tested the the fix using the two namings with the following code, and everything went fine : 

    [Custom]
    [Custom2Attribute]
    class Program
    {
        static void Main(string[] args) { }
    }

    public class CustomAttribute : Attribute { }
    public class Custom2Attribute : Attribute { }

Reply:
@sharwell What should I do now, to help getting a mergeable PR ? Did you find the tests for this feature ? Should I write them ?